### DISCUSSION TITLE
Asking for reverse polarity protection ideas for Bolt with 18650 li-ion battery

### DISCUSSION THREAD

**Discussion by tmartindev** [bitcraze/discussions#1167] [Q&A]
Hello,
We have made a drone like the [one you shared in your blog](https://www.bitcraze.io/2020/10/testing-crazyflie-bolt-and-1-cell-li-ion/)
The battery is firmly held in place with a [battery holder from Keystone](https://www2.mouser.com/ProductDetail/Keystone-Electronics/1044?qs=up89fPMRNqDbJ6cCmjKOVA%3D%3D). However, this battery holder does not protect from reverse polarity.
Although we have added clear visual cues for users to not put the battery the wrong way, we still have people making that mistake and burning the main board.

We do not want to change the battery holder to a more secure version because it would mean buying new batteries with button top which are a about 25% more expensive and less available where we are.

So I am asking the community if we can use some kind of circuit (or maybe a mechanism ?) to prevent this issue without significant loss in performance (voltage drop, flight duration,...).

**Comment by tobbeanton** [comment]
You could use a reverse polarity protection circuit, e.g. as explained [here](https://electronics.stackexchange.com/questions/552381/need-help-in-understanding-the-mosfet-reverse-polarity-protection-circuit). With a good PMOS with low rds-on it should not cause much energy loss.
